script/main.js

```diff
@@ -220,9 +228,14 @@ class App extends Component {
   }
 
   render() {
+    const versions = params.has('version') ?
+      params.get('version').split(',').map(getMajorVersion) :
```

Comment:
what are you thinking on multiple versions in the URL?

my thought is we'd have this in the url: `&version=6&version=5`

and in the JS here: `params.getAll('version').map(...`

Reply:
wfm